src/locales/de.json

```diff
@@ -0,0 +1,144 @@
+{
+  "consent.action-accept": "Zulassen",
+  "consent.action-reject": "Ablehnen",
+  "consent.description": "Geben Sie nur Berechtigungen frei, wenn Sie dieser Website oder App vertrauen. Sie müssen nicht alle Berechtigungen akzeptieren.",
+  "consent.privacy-policy-label": "Datenschutzrichtlinie",
+  "consent.remember-label": "Entscheidung für das nächste Mal merken. Die Anwendung wird nicht in der Lage sein, zusätzliche Berechtigungen ohne Ihre Zustimmung anzufordern.",
+  "consent.remember-tooltip": "meine Entscheidung merken",
+  "consent.requested-permissions-label": "Die Anwendung fordert Zugriff auf die folgenden Berechtigungen an:",
+  "consent.terms-of-service-label": "Nutzungsbedingungen",
+  "error.back-button": "Zurück",
+  "error.description": "Es ist ein Fehler aufgetreten mit folgender Nachricht:",
+  "error.support-email-link": "Wenn das Problem weiterhin besteht, kontaktieren Sie uns bitte unter <a>{contactSupportEmail}</a>",
+  "error.title": "Ein Fehler ist aufgetreten",
+  "error.title-internal-server-error": "Interner Serverfehler",
+  "error.title-not-found": "404 - Seite nicht gefunden",
+  "kratos-messages.1010001": "Anmelden mit Sicherheitsschlüssel",
```

Comment:
I would like to avoid kratos here because customers of Ory Network probably do not user kratos directly / know what it is:

- `selfservice.messages.[id]`?
- `identities.messages.[id]`?